_includes/footer.html

```diff
@@ -39,8 +39,19 @@
     </div>
   </div>
   <div class="bottom">
-    <a class="logo" href="{{ "/" | relative_url }}"><img src="{{ "/images/rook-logo-small.svg" | relative_url }}" alt="rook.io" /></a>
-    <span>&#169; {{ site.time | date: '%Y' }} Rook Authors. All rights reserved.</span>
+    <div class="grid-center">
```

Comment:
can you share a screenshot of what these changes look like when rendered?

Reply:
You could get this for free with Netlify 😉 But yes, of course.